libs/glog/glog-Natron.patch

```diff
@@ -24,3 +24,16 @@ diff -ur /Users/devernay/Development/blender/extern/glog/src/windows/port.h ./sr
  
  inline char* strerror_r(int errnum, char* buf, size_t buflen) {
  #ifdef FREE_WINDOWS
+--- CMakeLists.txt.orig	2022-08-08 10:34:58.000000000 -0400
```

Comment:
Change already included in `CMakeLists.txt`.

Reply:
same